ere-guests/stateless-validator/ethrex/sp1/src/main.rs

```diff
@@ -17,12 +18,35 @@ pub fn main() {
     // and just checking them again.
     input.blocks[0].header.hash = Default::default();
     let parent_hash = input.blocks[0].header.parent_hash;
+    let versioned_hashes_hash: Option<[u8; 32]> = input
```

Comment:
Whenever Ethrex supports OpenVM and Zisk, I'll extract Ethrex guest program as I did with Reth to avoid this repetition.